supervisor/api/__init__.py

```diff
@@ -413,7 +414,14 @@ async def addons_addon_info(request: web.Request) -> dict[str, Any]:
             try:
                 return await api_addons.info(request)
             except APIAddonNotInstalled:
-                return await api_store.addons_addon_info(request)
+                # Route to store/{addon}/info but add missing fields
+                return dict(
+                    await api_store.addons_addon_info_wrapped(request),
+                    state=AddonState.UNKNOWN,
+                    options=self.sys_addons.store.get(
+                        request.match_info.get("addon")
+                    ).options,
```

Comment:
If we know the keys are there we should use `dict[key]`.
```suggestion
                    options=self.sys_addons.store[
                        request.match_info["addon"]
                    ].options,
```